main_spec.tex

```diff
@@ -112,7 +112,7 @@ \subsubsection{\textbf{SHPALLOC}}\label{subsec:shpalloc}
 \subsubsection{\textbf{SHPCLMOVE}}\label{subsec:shpclmove}
 \input{content/shpclmove.tex}
 
-\subsubsection{\textbf{SHPDEALLC}}\label{subsec:shpdealloc}
+\subsubsection{\textbf{SHPDEALLOC}}\label{subsec:shpdealloc}
```

Comment:
The routine in question is actually called ```SHPDEALLC```, so the right fix here is to rename the input file rather than renaming the section.  Could you push a patch to fix this?